DOCS.md

```diff
@@ -75,21 +75,22 @@ module.exports = {
   plugins: [
     ["module-resolver", {
       alias: {
+        "foo": ([, name]) => path.join('bar', name)
         "^@namespace/foo-(.+)": ([, name]) => path.join('packages', name)
       }
     }]
   ]
 }
 ```
 
-Using the config from this example `'@namespace/foo-bar'` will become `'packages/bar'` or `'packages\\bar'` depending on the OS.
+Using the config from this example:
+* `'foo/baz'` will become `'bar/baz'` or `'bar\\baz'`
```

Comment:
Not sure if we should use `path.join` as an example.

On windows or unix, all paths in node for the import/require files should in theory be in a posix format afaik... right?

Reply:
AFAIK Node allows me to use `\` on Windows. I'll adjust the examples though, to make them more simple.

Reply:
On a second thought, manually joining path parts is a bad practice... I'd rather stick to `path.join` than use `+`.

Reply:
Yeah, I agree that in general people should always use `path.join` to create a path. But in this case, inside a node/web project, I don't think it's a good idea to mix posix and windows paths.

What if a project is build on windows and provide paths with `\`? Would it run on a mac/linux pc? Will webpack understand the paths?
I feel it's risky to have this in our doc since most people should never use windows paths for their imports/require.

Maybe replacing the example to remove path.join would be best.

Reply:
Ok, will remove them.